contracts/libraries/LibQuote.sol

```diff
@@ -147,21 +149,52 @@ library LibQuote {
 		}
 	}
 
+//	/**
+//	 * @notice Gets the trading fee for a quote.
+//	 * @param quoteId The ID of the quote for which to get the trading fee.
+//	 * @return fee The trading fee for the quote.
+//	 */
+//	function getTradingFee(uint256 quoteId) internal view returns (uint256 fee) {
+//		QuoteStorage.Layout storage quoteLayout = QuoteStorage.layout();
+//		Quote storage quote = quoteLayout.quotes[quoteId];
+//		if (quote.orderType == OrderType.LIMIT) {
+//			fee = (LibQuote.quoteOpenAmount(quote) * quote.requestedOpenPrice * quote.tradingFee) / 1e36;
+//		} else {
+//			fee = (LibQuote.quoteOpenAmount(quote) * quote.marketPrice * quote.tradingFee) / 1e36;
+//		}
+//	}
+
 	/**
-	 * @notice Gets the trading fee for a quote.
-	 * @param quoteId The ID of the quote for which to get the trading fee.
-	 * @return fee The trading fee for the quote.
+	 * @notice Gets the open fee for a quote.
+	 * @param quoteId The ID of the quote for which to get the open fee.
+	 * @return fee The open fee for the quote.
 	 */
-	function getTradingFee(uint256 quoteId) internal view returns (uint256 fee) {
+	function getOpenFee(uint256 quoteId) internal view returns (uint256 fee) {
 		QuoteStorage.Layout storage quoteLayout = QuoteStorage.layout();
 		Quote storage quote = quoteLayout.quotes[quoteId];
 		if (quote.orderType == OrderType.LIMIT) {
-			fee = (LibQuote.quoteOpenAmount(quote) * quote.requestedOpenPrice * quote.tradingFee) / 1e36;
+			fee = (LibQuote.quoteOpenAmount(quote) * quote.requestedOpenPrice * quote.tradingFee.openFee) / 1e36;
 		} else {
-			fee = (LibQuote.quoteOpenAmount(quote) * quote.marketPrice * quote.tradingFee) / 1e36;
+			fee = (LibQuote.quoteOpenAmount(quote) * quote.marketPrice * quote.tradingFee.openFee) / 1e36;
 		}
 	}
 
+	/**
+	 * @notice Gets the close fee for a quote.
+	 * @param quoteId The ID of the quote for which to get the close fee.
+	 * @return fee The close fee for the quote.
+	 */
+	function getCloseFee(uint256 quoteId) internal view returns (uint256 fee) {
```

Comment:
unused actually

Reply:
Given that the closeFee is actually not used anywhere, why are we introducing the breaking change from tradingFee to openFee and closeFee?